server/ansible/roles/pbench-server-disable/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Example: /var/www/html/pbench-results-host-info.versioned/pbench-results-host-info.URL002
+#          ^-----------^ ^-------------------pbench_host_info_prefix-------------------^
+#                ^
+#     httpd_document_root_dir
+- name: "set up links to the maintenance files"
+  file:
+    path: "{{ httpd_document_root_dir }}/{{ pbench_host_info_prefix }}{{ item }}"
+    src:  "{{ pbench_host_info_prefix | basename }}{{ item }}.maint"
+    state: link
+  with_items:
+    - "002"
+
+# we should wait until the queue is drained before disabling crontab
+# but there is no easy way to do that: pung and do it by hand?
+
+# - name: "disable crontab"
+#   command:
+#     cmd: crontan -u pbench -r
```

Comment:
`crontab`

Reply:
Fixed

Reply:
Another off-by-1 error - I refer you to my sig :)